docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   ghsci:
-    image: globalhealthyliveablecities/global-indicators:v4.7.2
+    image: ryntrnr/global-indicators-earth-engine:v4.5.2
```

Comment:
This is okay while we test (useful even!), but ultimately if we are to merge changes into our main branch (as per other comment) we need to build and push a GEE-specific docker image and launcher to isolate these changes from the regular process.  Ultimately, we can't have your ryntrnr dockerhub account merged in here!

Reply:
It looks like you might not have merged changes from 4.7.2 maybe though?  Might be why the Dockerfile has some older code?